app/javascript/utilities/useDisableGrammarly.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Disable grammarly if certain conditions are met
+ *
+ * @returns {object|undefined} properties to disable grammarly
+ *
+ * @example
+ * <textarea {...disabledGrammarlyProps} />
+ */
+export function useDisableGrammarly() {
```

Comment:
```suggestion
export function useDisableGrammarlyInChrome() {
```
small change (blocking) - I think we should reflect in the name that the blocking is only going to apply in Chrome